src/components/posts/PostListItemCard/PostListItemCard.tsx

```diff
@@ -57,45 +69,83 @@ const useStyles = makeStyles(({ spacing, typography }) => {
 
 type PostListItemCardProps = PostListItemCardFragment & {
   className?: string;
+  project: ProjectOverviewQuery['project'];
+  canEdit: boolean;
 };
 
 export const PostListItemCard: FC<PostListItemCardProps> = (props) => {
   const classes = useStyles();
   const formatDate = useDateTimeFormatter();
+  const [actionsAnchor, setActionsAnchor] = useState<MenuProps['anchorEl']>();
+  const [edit, setEdit] = useState(false);
+  const [deletePost, setDeletePost] = useState(false);
 
   return (
-    <Card className={clsx(classes.root, props.className)}>
-      <CardContent className={classes.cardContent}>
-        <div className={classes.leftContent}>
-          <Avatar className={classes.avatar}>
-            {props.creator.value?.avatarLetters}
-          </Avatar>
-        </div>
-        <div className={classes.rightContent}>
-          <div className={classes.topInfo}>
-            <Typography variant="body2">
-              {`${props.creator.value?.displayFirstName.value} ${props.creator.value?.displayLastName.value}`}
-            </Typography>
-            <Typography variant="body2" color="textSecondary">
-              {formatDate(props.createdAt)}
-            </Typography>
+    <>
+      <Card className={clsx(classes.root, props.className)}>
+        <CardContent className={classes.cardContent}>
+          <div className={classes.leftContent}>
+            <Avatar className={classes.avatar}>
+              {props.creator.value?.avatarLetters}
+            </Avatar>
           </div>
-          <div className={classes.typeShareabilityRow}>
-            <Typography variant="h4">{props.type}</Typography>
-            <div className={classes.shareability}>
-              <Typography variant="body2">
-                <span className={classes.shareabilityLabel}>
-                  {props.shareability.includes('External')
-                    ? 'PUBLIC'
-                    : 'PRIVATE'}
-                </span>
-                {displayPostShareability(props.shareability)}
-              </Typography>
+          <div className={classes.rightContent}>
+            <div className={classes.topRightContent}>
+              <div className={classes.nameAndDate}>
+                <Typography variant="body2">
+                  {`${props.creator.value?.displayFirstName.value} ${props.creator.value?.displayLastName.value}`}
+                </Typography>
+                <Typography variant="body2" color="textSecondary">
+                  {formatDate(props.createdAt)}
+                </Typography>
+              </div>
+              {props.canEdit && (
+                <IconButton onClick={(e) => setActionsAnchor(e.currentTarget)}>
+                  <MoreVert />
+                </IconButton>
```

Comment:
I think it would look better if this was made square to the card
![Screen Shot 2021-07-19 at 10 49 39 AM](https://user-images.githubusercontent.com/932566/126190857-c4df93bb-8622-46f6-96ad-d288cab76c6f.png)
![Screen Shot 2021-07-19 at 11 02 44 AM](https://user-images.githubusercontent.com/932566/126190953-d1c91591-f608-40bc-b864-98c733241485.png)